src/lambda/models/samTemplates.ts

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Runtime } from 'aws-sdk/clients/lambda'
+import { Set } from 'immutable'
+
+export const helloWorldTemplate = 'AWS SAM Hello World'
+export const eventBridgeHelloWorldTemplate = 'AWS SAM EventBridge Hello World'
+export const eventBridgeStarterAppTemplate = 'AWS SAM EventBridge App from Scratch'
```

Comment:
What do you think about surfacing this mode from the context menu of Registry/Schema nodes (whichever is more granular) in the AWS Explorer?

Reply:
I would personally preferred that option, would have cut number of extra steps introduced into the wizard.   However, we want this to be part of the new sam application flow, (feeling bad to say this infamous remark again) that is how it is implemented in Intellij plugin extension as well.  So, to keep the parity minimum I will keep it as it is.